src/core_plugins/region_map/public/region_map_vis_params.html

```diff
@@ -23,7 +23,7 @@
 
     <div class="form-group clearfix" ng-hide="!editorState.params.emsHotLink">
         <a
-          class="ems-hotlink pull-right"
+          class="rgmEMSLink pull-right"
```

Comment:
@cchaos Not for this PR, but do you think we should make an `eui-floatRight` utility for stuff like this, or do you think we should discourage floats completely?

Reply:
I'm super on the fence about that, we can talk offline.